src/views/UserSubmissions.js

```diff
@@ -170,59 +178,145 @@ class UserSubmissions extends Component {
 
   render() {
     const handleClose = () => this.setState({ showDeleteWarning: false });
-    return (
-      <div>
-        <div
-          style={{
-            display: 'flex',
-            justifyContent: 'center',
-            position: 'relative',
-            bottom: '100px',
-          }}
-        >
+
+    if (!this.state.isLoggedIn) {
+      return (
+        <div>
           <div
             style={{
               display: 'flex',
-              flexDirection: 'row',
-              justifyContent: 'space-around',
-              width: '50%',
+              justifyContent: 'center',
+              width: '100%',
             }}
           >
-            <LandingButton
-              variant="outlined"
-              type="button"
-              onClick={() => this.startSurvey()}
+            <div
+              style={{
+                display: 'flex',
+                flexDirection: 'row',
+                justifyContent: 'space-around',
+                width: '50%',
+              }}
             >
-              Start New Survey
-            </LandingButton>
-            <LandingButton variant="outlined" type="button" href={faqPath}>
-              FREQUENTLY ASKED QUESTIONS
-            </LandingButton>
-            <LandingButton variant="outlined" type="button" href={guidancePath}>
-              GUIDE LINK
-            </LandingButton>
+              <Signup signedOut={true} admin={true} />
```

Comment:
Go ahead and remove this. They can sign in via the log in link at the top

Reply:
This button here is different than the logged in one. If the user is logged out, this button will let them sign up. Added it in via this design,

Reply:
<img width="1168" alt="Screen Shot 2021-11-12 at 9 19 24 AM" src="https://user-images.githubusercontent.com/42750449/141481744-5308c4ea-5778-4452-bca6-400c53095abd.png">